src/server/middleware/blob/util.go

```diff
@@ -0,0 +1,40 @@
+package blob
+
+import (
+	"fmt"
+	"github.com/goharbor/harbor/src/lib/errors"
+	"github.com/goharbor/harbor/src/lib/log"
+	blob_models "github.com/goharbor/harbor/src/pkg/blob/models"
+	"github.com/goharbor/harbor/src/server/middleware/requestid"
+	"net/http"
+)
+
+// probeBlob handles config/layer and manifest status in the PUT Blob & Manifest middleware, and update the status before it passed into proxy(distribution).
+func probeBlob(r *http.Request, digest string) error {
+	ctx := r.Context()
+	logger := log.G(ctx)
```

Comment:
Delete `ctx := r.Context()` and use `log.G(r.Context())` otherwise use ctx direlcty in the following statements.

Reply:
updated.